easy_music_generator/easy_music_generator.py

```diff
@@ -11,45 +11,50 @@ def __init__(self):
         self.note_matrix = None
         self.chord_distribution = None
 
-
-    def generate(self, tempo=90, bars=4, output_path='/music_generator_output'):
+    def generate(self, tempo=90, bars=4,
```

Comment:
ignore pep8, or create the variable outside. 

filepath = '/music_generator_output'
def generate(self, tempo=90, bars=4, output_path=filepath):

Reply:
Okay! Good idea, made the change